### ISSUE TITLE
Containers instantly clear after being opened

### ISSUE THREAD

**Issue by ChilicookieZ** [cafestifflered/ContainerFaker#16]
I ran the clearcache command earlier, didn’t think it had worked as nothing happened (didn’t say anything in console either) now players are reporting that they will open a container, close it, reopen and it’s empty or instantly has new items (rather than having the 10 minute timer happen).

**Comment by cafestifflered** [comment]
Could not reproduce in test environment, but this may also be the intended behavior. When the cache is cleared, all of the containers on the server have their contents wiped and regenerated the next time the player opens the container.

This is a confirmed bug if the player can constantly reopen the same container and get new loot each time.

**Comment by Owen1212055** [comment]
Correct, still trying to figure out why this is happening.. if any reproduction/debug info could be given that would be great.
